taskcluster/ci/update-project/kind.yml

```diff
@@ -36,4 +38,12 @@ jobs:
     mozilla-mobile/focus-android:
         repo-prefix: focusandroid
         pr-target: main
-
+    mozilla-mobile/firefox-android:
+        repo-prefix: firefoxandroid
+        pr-target: ac-prep  # TODO: Change to main once we migrated android-components over
+        # Note: Path to `l10n.toml` includes a subfolder because the monorepo will contain
+        # one subfolder per android product
+        #
+        # TODO: Support more than a single product
+        l10n-toml-path: 'android-l10n/mozilla-mobile/android-components/l10n.toml'
+        destination-repo: 'mozilla-mobile/firefox-android/android-components'
```

Comment:
Since we want 2 config items per subdir, we may want to make this a dictionary?

```
targets-or-some-other-name:
    android-components:
        l10n-toml-path: 'android-l10n/mozilla-mobile/android-components/l10n.toml'
        destination-repo: 'mozilla-mobile/firefox-android/android-components'  # should we support staging here? '{owner}/{base_repo}/android-components' ?
        l10n-path-subdirectory: '{repo}/android-components'
        some-other-config: 'how-to-remove-/firefox-android/'
    focus-android:
        ...
```

which might make all these hacks more programmatic / config-driven?

Reply:
Good idea! I'll go with a dict once we port focus-android to the monorepo.